js-plugins/mcp/src/connect.ts

```
@@ -0,0 +1,58 @@
+import type { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
+import type { Server } from '@farmfe/core'
+import { SSEServerTransport } from '@modelcontextprotocol/sdk/server/sse.js'
+import { Context, Next } from "koa"
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)